src/events.c

```diff
@@ -69,7 +69,7 @@ void inform_slide_keyboard(gboolean value)
         GError* err = NULL;
         assert(gconfclient != NULL);
         if (!gconf_client_set_bool(gconfclient, SLIDE_OPEN_KEY,
-                                   !value, &err) && err != NULL) {
```

Comment:
@freemangordon, this is the change I mentioned in PR

Reply:
ah, so you have changed the meaning of the 'value' parameter. Did you verify that your code is the only user of that function?

Reply:
Yes, I did.